docs/reference/experiment-config.txt

```diff
@@ -659,6 +659,8 @@ large-scale experiments with hundreds or thousands of trials.
 Adaptive (Simple)
 =================
 
+Note: To be deprecated. Recommended to use state-of-the-art Adaptive (ASHA).
```

Comment:
Is it deprecated, or is it going to be deprecated?

> Recommended to use state-of-the-art Adaptive (ASHA).

Full sentences please :)

Reply:
This is a notice that it is going to be deprecated

Reply:
Why not just deprecate it now? i.e., to me, "deprecation" is itself a notice. A notice of an upcoming notice seems a little redundant, no?